app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def calculate_team_total_rating(players_data: list) -> int:
+    counter = 0
+    for player in players_data:
+        counter += player.get_rating()
+
+    return counter
+
+
+def elves_concert(elfs: list) -> None:
```

Comment:
Indicate that you expect to obtain a list of `Elf` class instances in the type annotation. Extend type annotations by specifying what instances you expect to obtain for each argument in other functions as well